lib/widgets/molecules/resources_section.dart

```diff
@@ -1,51 +1,57 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:mm_flutter_app/constants/app_constants.dart';
-import 'package:mm_flutter_app/widgets/atoms/resource_tile.dart';
 import 'package:mm_flutter_app/widgets/atoms/section_tile.dart';
 
+import 'image_tile.dart';
+
 class ResourcesSection extends StatelessWidget {
   const ResourcesSection({Key? key}) : super(key: key);
 
   List<Widget> _createResourceTiles(AppLocalizations l10n) {
     //TODO(m-rosario): Don't use hardcoded values.
     //TODO(m-rosario): Implement callback for each tile.
     List<Widget> resourceTiles = [
-      Components.resourceListItemDivider,
-      ResourceTile(
-        icon: const Icon(Icons.school),
-        text: l10n.homeResourcesMentorTraining,
-        onPress: () => {},
+      ImageTile(
+        imageUrl:
+            "https://freevector-images.s3.amazonaws.com/uploads/vector/preview/40309/YOUNG_GIRL_WORKING_WITH_LAPTOP.jpg",
```

Comment:
Generally, static images and similar content are stored in the source code as assets. This is a lot more efficient than downloading the same image every time this code is called. With avatars we use URL because these are stored externally, and could change at any time.

Reply:
Makes sense, changed to local assets